pkg/aws/ec2/ec2_test.go

```diff
@@ -188,13 +118,6 @@ func TestCollector_Collect(t *testing.T) {
 	})
 	t.Run("Collect should return a ClientNotFound Error if the ec2 client is nil", func(t *testing.T) {
 		ps := mockpricing.NewPricing(t)
-		ps.EXPECT().GetProducts(mock.Anything, mock.Anything, mock.Anything).
```

Comment:
It's just that the test was failing because this expects the function to be called once, and now we're returning the error before the function is called. Basically, it yelled at me XD